### ISSUE TITLE
Can't make more than one page grouped by "Name"

### ISSUE THREAD

**Issue by qwirksilver** [joyrider3774/Playnite_html_exporter#8]
## Steps to reproduce
1. In the add-on settings, leave page 1 set to the default: "Group field" set to "Name."
2. Add a new page and give it a name.
3. Set the new page's "Group field" to "Name."

## Expected
- The new page is not generated and does not appear in the list of pages in the top navigation of the exported site.

## Actual
- The new page should be generated and should appear in the list of pages in the top navigation of the exported site.

## Notes 
- Seems like something is maybe wrong with the relationship between the "Name" group and the generation of the set of pages/index file. So this issue may be related: https://github.com/joyrider3774/Playnite_html_exporter/issues/7

**Comment by joyrider3774** [comment]
Hey there was indeed a bug, pages with Name groupfield always got named index.html and i have a safety in my code to not generate the same file twice with same fields. I fix it now but it's not released yet.

However it does not make much sense to have multiple pages with Name as groupfield. As data is grouped (/sorted) first on the groupfield and then on the sort field. So the only thing you will be able to change with this ability now, is to order in which double game names appear

**Comment by qwirksilver** [comment]
Wow! 🤩 Fastest bug fixes ever!

> However it does not make much sense to have multiple pages with Name as groupfield

I can see what you mean, however, there is definitely a use case. 😄  Here's mine:
- I want to give people the option of how they view my games. I can accomplish this via your awesome templates - the default view with images lets people browse easily visually, whereas the text list or table views allows people to more quickly see more games at once along with details, without needing to open each game page.
- Therefore, I want one page like "Name (grid)" which I set to the default view, and one page like "Name (list)" which I set to the text list or table view. (I have similar views set up: "Feature (grid)", "Feature (list)", "Genre (grid)", "Genre (list)")
